client/state/themes/selectors.js

```diff
@@ -574,6 +576,34 @@ export function isThemePremium( state, themeId ) {
 }
 
 /**
+ * Whether a WPCOM theme given by its ID belongs to the Premium Squared bundle.
+ *
+ * @param  {Object} state   Global state tree
+ * @param  {Object} themeId Theme ID
+ * @return {Boolean}        True if the theme is in the Premium Squared bundle
+ */
+export function isPremiumSquaredTheme( state, themeId ) {
+	const theme = getTheme( state, 'wpcom', themeId );
+	return !! theme && includes( [ 'Automattic', 'WooThemes' ], theme.author );
+}
+
+/**
+ * Whether a WPCOM theme can be activated on a site.
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {String}  themeId Theme ID for which we check availability
+ * @param  {Number}  siteId  Site ID
+ * @return {Boolean}        True if the premium theme is available for the given site
+ */
+export function isPremiumThemeAvailable( state, themeId, siteId ) {
+	return !! (
```

Comment:
Do we still need the coercion? We're only using boolean predicates (`is...`, `has...`) here, so those should return `true` or `false`...